doc/ref/obsolete.xml

```diff
@@ -192,9 +192,7 @@ your specific use case.
 is an info class to display warnings when an obsolete variable is used.
 By default, these warnings are switched off since the info level for 
 this class is 0. Setting it to 1 will trigger warnings if &GAP; will 
-detect that an obsolete variable is used at runtime (this detection is
-possible, however, only for obsolete variables declared using 
-<C>DeclareObsoleteSynonym</C>). 
+detect that an obsolete variable is used at runtime.
```

Comment:
There is no limitation here being dropped. Rather, what happens is this: Some official documentation references to an unofficial and undocumented internal function `DeclareObsoleteSynonym`, which it should not do. I then tried to rephrase the parens to not use it, but that made it realize that it actually does not convey much if any useful information to the user; in summary, the only thing the user might learn is that sometimes, we might fail to detect use of an obsolete, for reasons the user has no chance to understand ("so what is this DeclareObsoleteSynonym? And why don't they always use it?).

However, re-reading it again now, I realize another issue with this paragraph: It refers to "variables", which is highly misleading. Sure, from a purely technical point of view, it is correct, because things like `Group` and `IsGroup` are strictly speaking variables which contain or reference functions, filters, etc. 

But a user may very well read this as "global variable". Which is ironic, because that's the only kind of obsolete object we currently cannot warn about it being used.

So, I think this paragraph should instead refer to "functions" (or perhaps "functions, operation, properties, attributes, filters, ..." ?!).